virridy/verification_workflow/1_verification_setup.Rmd

```diff
@@ -42,7 +42,7 @@ get_working_data_decision()
 network <- "virridy"
 # Set the site and parameter to verify
 site <- "archery"
-parameter <- "Temperature"
+parameter <- "Chl-a Fluorescence"
 ```
 
 ```{r set the site-param df to update}
```

Comment:
Throwing error from chunk 5 "Error in UseMethod("group_by") :
no applicable method for 'group_by' applied to an object of class "NULL""

Reply:
Chunk 5 (set the site-param df to update) is giving you an error that has "group_by" in it?